commons/src/test/java/org/sonarsource/analyzer/commons/RuleMetadataLoaderTest.java

```diff
@@ -233,6 +235,34 @@ public void getStringArray() throws Exception {
     assertThat(RuleMetadataLoader.getStringArray(map, "key")).containsExactly("x", "y");
   }
 
+  @Test
+  public void getScopeIfPresent() {
+    RuleScope scope;
+
+    assertThat(RuleMetadataLoader.getScopeIfPresent(Collections.emptyMap(), "scope")).isNotPresent();
+    assertThat(RuleMetadataLoader.getScopeIfPresent(Collections.singletonMap("scope", null), "scope")).isNotPresent();
+    assertThat(RuleMetadataLoader.getScopeIfPresent(Collections.singletonMap("scope", false), "scope")).isNotPresent();
+
+    scope = RuleMetadataLoader.getScopeIfPresent(Collections.singletonMap("scope", "main"), "scope").orElse(null);
+    assertThat(scope).isEqualTo(RuleScope.MAIN);
```

Comment:
I think `orElse(null)` is unnecessary, you can use `assertThat(scope).contains(...)` to assert directly optional value

Reply:
Thanks for pointing this out. I have updated accordingly.